config/application.rb

```diff
@@ -253,5 +253,6 @@ module ::Reporting; end
     config.exceptions_app = self.routes
 
     config.view_component.generate.sidecar = true # Always generate components in subfolders
+    config.flipper.test_help = false
```

Comment:
May I suggest an alternative solution?

We are doing our test setup before the whole test suite runs:
https://github.com/openfoodfoundation/openfoodnetwork/blob/5f302f347a99dd4119decd97aaa66edaa1e7ef80/spec/base_spec_helper.rb#L99-L103

If we just call the FeatureToggle.setup! before each test run then that may solve the problem, too. And then we are working with the default test config for Flipper.

Reply:
Hi @mkllnk 
replace `suite` with `each`, is that what you're asking me to do?
```ruby
 config.before(:each) do 
   Flipper.features.each(&:remove) 
   OpenFoodNetwork::FeatureToggle.setup! 
 end 
```
if yes, it's not working.

Reply:
I still don't know what's happening here. The specs pass locally but fail on CI. Maybe it's about KnapsackPro? I don't know how they implemented the queue.

Resetting Flipper seems to happen at the right time. But maybe KnapsackPro starts a new thread which uses a new Flipper instance with a new memory adapter? It doesn't make sense to me.